modules/nw-install-config-gateway-api.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * networking/about-contour-operator.adoc
+
+[id="nw-install-config-gateway-api_{context}"]
+= Installing and configuring gateway API
+
+The following guide provides instructions for using link:https://gateway-api.sigs.k8s.io/[Gateway API] with the Contour Operator on {product-title}.
+
+.Prerequisites
+
+* You installed an {product-title} cluster and the `kubectl` command line.
```

Comment:
xref https://github.com/openshift/openshift-docs/pull/33631/files#r656413438 regarding `kubectl`.

Reply:
Do these command translate directly to `oc`?